### ISSUE TITLE
PVS-7 on/off options on the PASGT/Bump helmets

### ISSUE THREAD

**Issue by EnsignOnDeck** [sharkster91/SMUI#54]
Unlike the individual PVS-5s, the PASGT w/ PVS-7's don't act as night vision no matter what I've done. It only allows me to flip it up and down and set the power. When I go to turn it on it only uses my lighter and it doesn't do anything when I drop my lighter and walk away (other light sources included).

Can't put in any batteries either and clicking the little drop down to see the remaining batteries, it doesn't show anything.

tl;dr night vision on helmets don't work.
